python/src/lib.rs

```diff
@@ -150,12 +150,12 @@ fn glide(_py: Python, m: &PyModule) -> PyResult<()> {
         let len = iterator.len();
 
         iterator.try_fold(Vec::with_capacity(len), |mut acc, val| {
-            acc.push(redis_value_to_py(py, val)?);
+            acc.push(resp_value_to_py(py, val)?);
```

Comment:
Isn't `resp` still redis? Should it be `valkey_value_to_py` instead?

Reply:
yes - but valkey currently uses the resp protocols. so we don't have anything better yet.

Reply:
valkey uses the same protocol as redis, which is RESP.